mongo_connector/command_helper.py

```diff
@@ -16,59 +16,29 @@
 """
 
 import logging
-import mongo_connector.errors
+from mongo_connector.dest_mapping import DestMapping
 
 LOG = logging.getLogger(__name__)
 
 
 class CommandHelper(object):
-    def __init__(self, namespace_set=[], dest_mapping={}):
-        self.namespace_set = namespace_set
-        self.dest_mapping = dest_mapping
-
-        # Create a db to db mapping from the namespace mapping.
-        db_pairs = set((ns.split('.')[0],
-                        self.map_namespace(ns).split('.')[0])
-                       for ns in self.namespace_set)
-        targets = set()
-        for _, dest in db_pairs:
-            if dest in targets:
-                dbs = [src2 for src2, dest2 in db_pairs
-                       if dest == dest2]
-                raise mongo_connector.errors.MongoConnectorError(
-                    "Database mapping is not one-to-one."
-                    " %s %s have collections mapped to %s"
-                    % (", ".join(dbs),
-                       "both" if len(dbs) == 2 else "all",
-                       dest))
-            else:
-                targets.add(dest)
-
-        self.db_mapping = {}
-        for src, dest in db_pairs:
-            arr = self.db_mapping.get(src, [])
-            arr.append(dest)
-            self.db_mapping[src] = arr
+    def __init__(self, dest_mapping_stru=DestMapping([],[],{})):
```

Comment:
Don't use mutable objects as default values for keyword arguments. See http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments. Here, it seems that "dest_mapping_stru" is actually a required argument, because I don't see any places where we don't pass an instance of DestMapping, so it doesn't need a default value.